src/views/AssistantPage.vue

```diff
@@ -182,10 +187,7 @@ export default {
 					this.task.completionExpectedAt = updatedTask.completionExpectedAt
 					this.task.scheduledAt = updatedTask.scheduledAt
 
-					const setProgress = (progress) => {
-						this.progress = progress
-					}
-					pollTask(updatedTask.id, setProgress).then(finishedTask => {
+					pollTask(updatedTask.id, this).then(finishedTask => {
```

Comment:
```suggestion
					pollTask(updatedTask.id, this, this.updateTask).then(finishedTask => {
```

You don't pass `this.updateTask` to pollTask here. Is it intentional? It seems like it should be passed.
The progress is not set in the assistant page if one schedules a task A, switches to another task B and loads the task A again before it has started.